e2e/test-helpers/themes.ts

```diff
@@ -0,0 +1,11 @@
+export const themes = [
+  'light',
```

Comment:
I wonder if we can directly get the primitives? Like [this](https://github.com/primer/react/blob/main/src/theme-preval.js#L5) Please ignore if you were already planning or this is just for a start :)